featuretools/variable_types/variable.py

```diff
@@ -44,7 +44,7 @@ def __eq__(self, other):
             self.entity_id == other.entity_id
 
     def __repr__(self):
-        ret = u"<Variable: {} (dtype = {})>".format(self.name, self.dtype)
+        ret = u"<Variable: {} >".format(self.name)
```

Comment:
still print the dtype, just use `self._dtype_repr`

Reply:
Fixed.